packages/puppeteer-core/src/common/ExecutionContext.ts

```diff
@@ -47,25 +47,6 @@ const getSourceUrlComment = (url: string) => {
 };
 
 /**
- * Represents a context for JavaScript execution.
```

Comment:
Keep comments.

Reply:
This is left-over from when this was public. The comment is not relevant anymore (and we plan to remove this anyway).